sdk_experimental/test/opentelemetry/sdk/trace/propagation/trace_context/response_text_map_propagator_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator do
+  let(:traceresponse_key) { 'traceresponse' }
+  let(:propagator) do
+    OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator.new
+  end
+  let(:valid_traceresponse_header) do
+    '00-000000000000000000000000000000AA-00000000000000ea-01'
+  end
+  let(:invalid_traceresponse_header) do
```

Comment:
This is not used AFAICT

Reply:
Right, it used to be used in the extract tests, but the functionality got removed.